packages/core/src/components/legend-item/legend-item.tsx

```diff
@@ -0,0 +1,111 @@
+import { Component, ComponentInterface, Element, Event, EventEmitter, h, Host, Prop, State } from "@stencil/core";
+import clsx from "clsx";
+
+/**
+ * @slot - The label for this legend item. Omit the label when a `dso-selectable` is placed in slot `selectable` is
+ * present. The label should than be on the `dso-selectable`
+ * @slot symbol - A span where the symbol is styled upon
+ * @slot selectable - A dso-selectable for this legend item
+ * @slot body - The slot to place controls in (i.e. `dso-input-range` or multiple `dso-selectable`\`s)
+ */
+@Component({
+  tag: "dso-legend-item",
+  styleUrl: "legend-item.scss",
+  shadow: true,
+})
+export class LegendItem implements ComponentInterface {
+  @Element()
+  host!: HTMLDsoLegendItemElement;
+
+  /**
+   * To disable the Legend Item
+   */
+  @Prop({ reflect: true })
+  disabled = false;
+
+  /**
+   * Shows a trash-can that, when clicked, emits `dsoRemoveClick`.
+   */
+  @Prop()
+  removable?: boolean;
+
+  /**
+   * Emitted when the user activates the remove button.
+   */
+  @Event()
+  dsoRemoveClick!: EventEmitter<MouseEvent>;
+
+  /**
+   * Emitted when the mouse enters the Legend Item
+   */
+  @Event()
+  dsoMouseEnter!: EventEmitter;
+
+  /**
+   * Emitted when the mouse leaves the Legend Item
+   */
+  @Event()
+  dsoMouseLeave!: EventEmitter;
+
+  @State()
+  showBody = false;
+
+  get symbolSlottedElement() {
+    return this.host.querySelector("[slot='symbol']");
+  }
+
+  get selectableSlottedElement() {
+    return this.host.querySelector("[slot='selectable']");
+  }
+
+  get bodySlottedElement() {
+    return this.host.querySelector("[slot='body']");
+  }
+
+  render() {
+    const hasSymbol = this.symbolSlottedElement !== null;
+    const hasBody = this.bodySlottedElement !== null;
+    const isSelectable = this.selectableSlottedElement !== null;
+
+    return (
+      <Host onMouseEnter={() => this.dsoMouseEnter.emit()} onMouseLeave={() => this.dsoMouseLeave.emit()}>
+        <div
+          class={clsx(
+            "dso-legend-item",
+            {
+              "dso-legend-item-symbol": hasSymbol,
+            },
+            {
+              "dso-legend-item-selectable": isSelectable,
+            },
+          )}
+        >
+          <div hidden={!hasSymbol}>
+            <slot name="symbol" />
+          </div>
+          <div hidden={!isSelectable}>
+            <slot name="selectable" />
+          </div>
+          <span>
+            <slot></slot>
```

Comment:
Wat wordt er in dit anonieme slot geplaatst?

Reply:
Het label van het Legend-item, of niks in het geval er een selectable in de slot selectable zit, want dan zit het label op de selectable. De gebruiker moet het maar weten 🤷‍♂️

Reply:
Aha. Ik begrijp het iets beter.

Tijdens het reviewen vond ik dit onderdeel moeilijk kon doorgronden. Dat komt onder andere omdat het slotted element "symbol" voorwaardelijk wordt getoond, maar het anonieme slot wordt altijd getoond.

Maar nu leer ik dat het anonieme slot ook leeggelaten kan worden 🤔

Kunnen die twee gecombineerd worden? Dat maakt de API van dit component vriendelijker.

Reply:
Kun je hier een voorzetje geven?